askbot/deps/django_authopenid/views.py

```diff
@@ -178,7 +181,7 @@ def logout_page(request):
         'page_class': 'meta',
         'have_federated_login_methods': util.have_enabled_federated_login_methods()
     }
-    return render(request, 'authopenid/logout.html', Context(data))
```

Comment:
Thanks for the pull request. The Context() was added to support Django templates for those who really want it.